sdk/daml-lf/ledger-api-value/src/main/protobuf/com/daml/ledger/api/scalapb/package.proto

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+//
+// Proprietary code. All rights reserved.
```

Comment:
This doesn't look standard

Reply:
hmm, it doesn't indeed:
https://github.com/digital-asset/daml/blob/4ba35794d27d7e7bcdf99fdfd0e370e78f3c620a/sdk/canton/community/ledger-api/src/main/protobuf/com/daml/ledger/api/scalapb/package.proto#L2

Reply:
I have had a look at this again. This code desn't have DA license in it, because it is not a DA snippet. It is a standard phrasing of the package.proto. I will do the same in my file

Reply:
Corrected